README.md

```diff
@@ -5,4 +5,4 @@ A text-based adventure/rpg game
 This is my first larger project using python. It is to create a text-based adventure/rpg game called 'Adventure Colussus' i have already layed some of the foundations for the game
 but there is still much to do. I am looking forward to adding classes for both the enemies and the character you play as including saving your character using JSON. I would also like to do is to try not to hardcode my adventure. I want to make something where you pass a file that stores information about your adventure and automatically takes the user through it. A fellow redditor said YAML or JSON would be good for this, therefore i would like to give it a shot. So if anyone want to make any contributions, feel free. But other then that, happy coding! :D
 
-Edit: We now have 4 people working on the project (which for me is really cool) so yeah, shoutout to them: robotlightsyou, wireboy5 and ssguitarist
```

Comment:
SkepticalPotato2K: From your main branch at the command line pull Slow-Scallions main branch. Then checkout your add license branch, and merge your local main into add license. This will create the conflict we see here on your local machine, where you can resolve it and recommit, and then resubmit the pull request. Here is that process on a mac, I believe everything is the same for PC.

SP2k(main)$ git set-remote upstream https://github.com/Slow-Scallion4183/Adventure-Colussus
SP2k(main)$ git pull upstream main
-- this will probably error and ask you to set a config method, I believe ff-only is the safest, that's what I default to.
SP2k(main)$ git --config --ff-only
--more info https://stackoverflow.com/questions/62653114/how-to-deal-with-this-git-warning-pulling-without-specifying-how-to-reconcile
SP2k(main)$ git pull upstream main
SP2k(main)$ git checkout add_license
SP2k(add_license)$ git merge main
--here you will be presented with the conflict to resolve, I think. This will also be the time to fix the other conflict.
SP2k(add_license)$ git add
SP2k(add_license)$ git commit
SP2k(add_license)$ git push -u origin add_license

Then you would visit the website and reissue the updated pull request.